### PR TITLE
git-setup: add German translation

### PR DESCRIPTION
**Pull request by 3voltAmperes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/git-setup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# git setup
+
+> Erstelle ein Git Repositorium in einem bestimmten Verzeichnis und committe alle Dateien.
```
Comment (pixelcmtd) [inline pages.de/common/git-setup.md:3]: the other pages don't translate it either
```suggestion
> Erstelle ein Git Repository in einem bestimmten Verzeichnis und committe alle Dateien.
```

File: pages.de/common/git-setup.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+> Teil der `git-extras`.
+> Weitere Infos: <https://github.com/tj/git-extras/blob/master/Commands.md#git-setup>.
```
Comment (pixelcmtd) [inline pages.de/common/git-setup.md:5]:
```suggestion
> Weitere Informationen: <https://github.com/tj/git-extras/blob/master/Commands.md#git-setup>.
```

File: pages.de/common/git-setup.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- Erstelle ein Git Repositorium im aktuellen Verzeichnis und committe alle Dateien:
```
Comment (pixelcmtd) [inline pages.de/common/git-setup.md:7]:
```suggestion
- Erstelle ein Git Repository im aktuellen Verzeichnis und committe alle Dateien:
```

File: pages.de/common/git-setup.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+`git setup`
+
+- Erstelle ein Git Repositorium in einem bestimmten Verzeichnis und committe alle Dateien:
```
Comment (pixelcmtd) [inline pages.de/common/git-setup.md:11]:
```suggestion
- Erstelle ein Git Repository in einem bestimmten Verzeichnis und committe alle Dateien:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]
No unmentioned fixes found, approved.
Thanks for a contribution! 🚀

File: pages.de/common/git-setup.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages.de/common/git-setup.md:3]: Interesting... Why? 😄 🤔

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**